README.md

```diff
@@ -1,3 +1,88 @@
-pat
-===
-[![Build Status](https://travis-ci.org/gorilla/pat.png?branch=master)](https://travis-ci.org/gorilla/pat)
+# pat [![Build Status](https://travis-ci.org/gorilla/pat.png?branch=master)](https://travis-ci.org/gorilla/pat) [![Go Report Card](https://goreportcard.com/badge/github.com/gorilla/pat)](https://goreportcard.com/report/github.com/gorilla/pat)
```

Comment:
Copy the badges used for gorilla/mux - https://github.com/gorilla/mux - SVG icons for GoDoc and TravisCI